<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>1250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>1070</y>
      <width>471</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вычислить</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btnZagruz">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>1000</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Загрузить фотографии с разметкой колосьев пшеницы на Ваш компьютер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSquareUpload">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSquare">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Файл успешно загружен</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPhotoUpload">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>840</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPhoto">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>930</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фото успешно загружены</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblSquare_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>1081</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Загрузите файл с координатами вершин поля или архив, содержащий файлы с координатами вершин поля (.txt, .zip)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblSquare_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>821</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Загрузите архив, содержащий фотографии участков поля (.zip)&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;br/&gt;Имена загружаемых фото должны быть составлены по следующему принципу:&lt;/p&gt;&lt;p&gt;Foto_..._..._..._..._... (возможные форматы фото: .png, .jpg)&lt;/p&gt;&lt;p&gt;Первое число - номер поля, к которому относится эта фотография&lt;/p&gt;&lt;p&gt;Второе число - номер фотографии (для каждого поля нумерация начинается с 1)&lt;/p&gt;&lt;p&gt;Третье число - координата Х точки, в которой сделано фото. Дана в метрах.&lt;/p&gt;&lt;p&gt;Четвёртое число - координата Y точки, в которой сделано фото. Дана в метрах.&lt;/p&gt;&lt;p&gt;Пятое число - высота квадрокоптера над землёй в момент фотографирования. Дана в САНТИМЕТРАХ.&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Пример: Foto_1_4_202_330_223.png&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>771</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToMinimumContentsLength</enum>
    </property>
    <item>
     <property name="text">
      <string>Файл с координатами поля и фото участков поля заргужаются отдельно</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Файл с координатами поля и  фото участков поля загружаются в одном архиве</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnInfo">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>10</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Информация</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
